salt/config/__init__.py

```diff
@@ -2512,7 +2520,7 @@ def syndic_config(
     ]
     for config_key in ("log_file", "key_logfile", "syndic_log_file"):
         # If this is not a URI and instead a local path
```

Comment:
I suggest moving this comment into the docstring of the new "should_prepend_root_dir" function.